components/home/sessoes/equipe/MembroInfo.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="membro-info">
+    <a :href="membro.url_github" target="_blank">
+      <img :src="require('~/assets/avatar/' + membro.imagem)" :alt="membro.nome" :title="membro.nome">
+    </a>
+  </div>
+</template> 
+<script>
+export default {
+  name: 'MembroInfo',
+  props: {
+    membro: {
+      type: Object,
+      default: null
+    }
+  }
+}
+</script>
+<style lang="scss" scoped>
+.membro-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
```

Comment:
Adiciona isso aqui pra animação ficar mais suave:

```suggestion
transition: all 0.3s;
}
```